Formula/s-nail.rb

```diff
@@ -0,0 +1,75 @@
+class SNail < Formula
+  desc "Fork of Heirloom mailx."
+  homepage "https://www.sdaoden.eu/code.html"
+  url "https://www.sdaoden.eu/downloads/s-nail-14.9.3.tar.gz"
+  sha256 "9048abe94c8b732ddefcd70b9f6052da16977de76b07598790e4c763e97f911d"
+
+  # Define options
+  option "with-maximal", "Anything on, including experimental features."
+  option "with-netsend", "Sending messages directly to the mail provider via the SMTP protocol, instead of requiring a local mail-transfer-agent (MTA) who does."
+  option "with-medium", "Possibly what people want who need nothing but a MIME-capable mailx(1)."
+  option "with-minimal", "This is the most plain mailx(1)-alike mode, but with MIME support."
+  option "with-nulli", "Anything that can be turned off is off but iconv(3)."
+  option "with-null", "Anything that can be turned off is off."
+
+  # Formula depends on OpenSSL per default and in MAXIMAL and NETSEND
+  depends_on "openssl" if build.with?("maximal") || build.with?("netsend") || (build.without?("medium") && build.without?("minimal") && build.without?("nulli") && build.without?("null"))
+  # Formula depends on IDNA per default and in MAXIMAL, NETSEND and MEDIUM
+  depends_on "libidn" if build.with?("maximal") || build.with?("netsend") || build.with?("medium") || (build.without?("minimal") && build.without?("nulli") && build.without?("null"))
+
+  def install
+    args = %W[
+      VAL_PREFIX=#{prefix}
+      VAL_PRIVSEP_USER=#{ENV["USER"]}
```

Comment:
If the username has to be compiled in at build time, then this formula cannot be bottled, which would mean it cannot be included in homebrew/core. Is there a way to override this after it's already built?